guides/guide-for-adding-windows-node.md

```diff
@@ -90,57 +95,112 @@ kube-system kube-flannel-ds-54954 1/1 Running 0 1m
 Now you can add Windows-compatible versions of Flannel and kube-proxy. In order to ensure that you get a compatible version of kube-proxy, you'll need to substitute the tag of the image. The following example shows usage for Kubernetes v1.24.3, but you should adjust the version for your own deployment.
 
 ```bash
-curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml | sed 's/VERSION/v1.24.3/g' | kubectl apply -f -
-kubectl apply -f https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay.yml
+# Make sure to modify the version of kube-proxy image to match your kubernetes version
+# before applying it
+curl -L https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/hostprocess/flannel/kube-proxy/kube-proxy.yml | sed 's/VERSION/v1.25.3/g' | kubectl apply -f -
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/hostprocess/flannel/flanneld/flannel-overlay.yml
 ```
 
->  **Note** If you are using another version of kubernetes on your windows node, change v1.24.3 with your own version .
+>  **Note** If you are using another version of kubernetes on your Windows node, change v1.24.3 with your own version .
 > To find your version of kubernetes run the following command:
 > `kubeadm version`
 
 5. Apply kube-flannel-rbac.yml from sig-windows-tools/kubeadm/flannel
 Next you will need to apply the configuration that allows flannel to spawn pods and keep them running:
 
 ```bash
-git clone https://github.com/kubernetes-sigs/sig-windows-tools
-kubectl apply -f sig-windows-tools/kubeadm/flannel/kube-flannel-rbac.yml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/kubeadm/flannel/kube-flannel-rbac.yml
+```
+
+
+## Configuring Calico hostprocess
+
+>  **Note:** All code snippets in Linux sections are to be run in a Linux environment on the Linux worker node.
+
+1. Prepare Kubernetes control plane for Calico
+
+Some minor preparation is recommended on the Kubernetes control plane in our cluster. It is recommended to enable bridged IPv4 traffic to iptables chains when using Flannel. The following command must be run on all Linux nodes:
```

Comment:
`when using Flannel` in the Calico topic

Reply:
Forgot to modify it to calico from flannel